theme/mixins/focus.less

```diff
@@ -1,7 +1,7 @@
 .focus() {
   outline: none;
   z-index: 1;
-  box-shadow: 0 0 0 @field-border-width @sk-focus;
+  box-shadow: 0 0 0 2px fade(@sk-focus, 75%);
```

Comment:
Shouldn't we be changing `@sk-focus` directly, rather than changing the usage of it? i.e. `@sk-focus: fade(#1e90ff, 75%);`